mindmaps-graql/src/main/java/io/mindmaps/graql/internal/analytics/MaxMapReduce.java

```diff
@@ -17,7 +17,7 @@
 public class MaxMapReduce extends MindmapsMapReduce<Number> {
 
     public static final String MEMORY_KEY = "max";
-    public static final String SELECTED_DATA_TYPE = "SELECTED_DATA_TYPE";
+    private static final String SELECTED_DATA_TYPE = "SELECTED_DATA_TYPE";
```

Comment:
Maybe this string should be moved up to `MindmapsMapReduce` since it is defined in both `MaxMapReduce` and `MeanMapReduce`